src/components/books/index.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
```

Comment:
- [ ] Kindly add a book component that would display data on a single book. It should take the book data as a prop and renders the title and the author.

Reply:
- [ ] In addition, the single book component should also contain a button for removing the book.